peachjam/forms.py

```diff
@@ -224,7 +224,7 @@ def filter_queryset(self, queryset, exclude=None, filter_q=False):
             queryset = queryset.filter(taxonomies__topic__slug__in=taxonomies)
 
         if filter_q and q and exclude != "q":
-            queryset = queryset.filter(title__icontains=q)
+            queryset = queryset.filter(Q(title__icontains=q) | Q(citation__icontains=q))
```

Comment:
If the field is "Cap. 7" and I search for "cap 7", it won't match. I think we should separate search terms with `%`, chatgpt suggests:

```python
from django.db.models import Q

def search_name(queryset, search_string):
    terms = search_string.split()
    q_objects = Q()
    for term in terms:
        q_objects &= Q(name__icontains=term)
    return queryset.filter(q_objects)
```

Reply:
added